lib/streamlit/testing/v1/app_test.py

```diff
@@ -64,9 +64,12 @@
 )
 from streamlit.testing.v1.local_script_runner import LocalScriptRunner
 from streamlit.testing.v1.util import patch_config_options
+from streamlit.web.bootstrap import _fix_matplotlib_crash
 
 TMP_DIR = tempfile.TemporaryDirectory()
 
+_fix_matplotlib_crash()
```

Comment:
nit: Have you considered calling this in `AppTest.__init__()`? Not sure if it has any benefits, but feels slightly safer to only call this if a test is actually created

Reply:
I considered it, but I'm pretty sure it's fine, since you can turn it off with a config option if you really need it, and if anything I'd be more concerned about weirdness from trying to call it multiple times, while the module level execution does it just once.